pageserver/src/metrics.rs

```diff
@@ -105,31 +105,44 @@ pub(crate) static VEC_READ_NUM_LAYERS_VISITED: Lazy<Histogram> = Lazy::new(|| {
 });
 
 // Metrics collected on operations on the storage repository.
+#[derive(
+    Clone,
+    Copy,
+    enum_map::Enum,
+    strum_macros::EnumString,
+    strum_macros::Display,
+    strum_macros::IntoStaticStr,
```

Comment:
```suggestion
    IntoStaticStr,
```
the macro is already imported above.